### PR TITLE
Feat/1207949918221283 async update

### PR DESCRIPTION
**Pull request by sfffaaa**
1. Add some tools
2. Extract some variable
3. Refine

### CODE REVIEW

File: tools/constants.py [hunk 1]
```diff
@@ -0,0 +1 @@
+BLOCK_GENERATIME_TIME = 6
```
Comment (DocteurPing) [inline tools/constants.py:1]: Is this a typo or is it made on purpose ? Shouldn't it be `BLOCK_GENERATE_TIME` or `BLOCK_GENERATION_TIME` ?

File: tools/peaq_eth_utils.py [hunk 2]
```diff
@@ -107,10 +109,34 @@ def calculate_evm_default_addr(sub_addr):
     return Web3.to_checksum_address(new_addr.lower())
 
 
+# def sign_and_submit_evm_transaction(tx, w3, signer):
```
Comment (DocteurPing) [inline tools/peaq_eth_utils.py:112]: Can we delete this comment ?

File: tools/force_set_current_code.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    batch.compose_sudo_call(
+        'Paras',
+        'force_set_current_code',
+        {'para': args.para, 'new_code': data}
+    )
+    receipt = batch.execute()
+    print(f'receipt is success: {receipt.is_success}')
+
+    # target_block_num = block_num + 10
+    # batch = ExtrinsicBatch(substrate, KP_GLOBAL_SUDO)
```
Comment (DocteurPing) [inline tools/force_set_current_code.py:53]: Can we delete this comment ?

File: tools/utils.py [hunk 4]
```diff
@@ -43,6 +43,7 @@
 # ETH_URL = 'http://127.0.0.1:9933'
 AUTOTEST_URI = os.environ.get('AUTOTEST_URI')
 ETH_TIMEOUT = 6 * 5
+BLOCK_GENERATIME_TIME = 6
```
Comment (DocteurPing) [inline tools/utils.py:46]: Why do we have a second version of this variable ? is it possible that we use the one in constant file or is there any blocker for that ?

File: tools/constants.py (hunk 1 above)
Comment (sfffaaa) [reply tools/constants.py]: Updated

File: tools/force_set_current_code.py (hunk 3 above)
Comment (sfffaaa) [reply tools/force_set_current_code.py]: Updated

**Review by DocteurPing — APPROVED** [review]

**Pull request merged**